src/components/Reportees/index.jsx

```diff
@@ -1,46 +1,58 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
-import DrilldownIcon from '../DrilldownIcon';
+import CONFIG from '../../config';
 
 const styles = ({
-  root: {
-    margin: '0 0.5rem 0 0',
+  row: {
+    display: 'grid',
+    // Suggestions welcome; This will only work if you have 3 columns of data
+    gridTemplateColumns: '200px 6rem 6rem',
   },
-  header: {
-    margin: '0.5rem 0 0.5rem 0',
+  headerLabels: {
+    fontWeight: 'bold',
+    textAlign: 'center',
   },
-  icon: {
-    fontSize: '1rem',
-    verticalAlign: 'bottom',
+  name: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
   },
-  person: {
-    // This makes it line up better with the table of components
-    padding: '1px',
-    display: 'flex',
+  link: {
+    textAlign: 'center',
   },
 });
 
 const sortByPersonName = (a, b) => (a.cn <= b.cn ? -1 : 1);
 
 const Reportees = ({
-  classes, ldapEmail, partialOrg, onPersonDetails,
+  classes, ldapEmail, partialOrg, metrics,
 }) => (
-  <div className={classes.root}>
-    <div height="1rem">&nbsp;</div>
+  <div>
```

Comment:
You can use a material UI table to display the information.

https://material-ui.com/demos/tables/#tables